src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/SID.cs

```diff
@@ -336,19 +337,16 @@ private void CreateFromParts(IdentifierAuthority identifierAuthority, int[] subA
             if (identifierAuthority < 0 ||
                 (long)identifierAuthority > MaxIdentifierAuthority)
             {
-                throw new ArgumentOutOfRangeException(
-nameof(identifierAuthority),
-                    identifierAuthority,
-                    SR.IdentityReference_IdentifierAuthorityTooLarge);
+                throw new ArgumentOutOfRangeException(nameof(identifierAuthority), identifierAuthority, SR.IdentityReference_IdentifierAuthorityTooLarge);
```

Comment:
While the nameof was previously indented wrong, I liked the previous, chopped, style better.  This line now causes horizontal scrolling on the default github window width.

Reply:
I've reformatted it. In general i'm not a fan of limiting things because github has a silly limitation though because it feels like limiting line lengths to 72 chars because of terminals.

Reply:
GitHub's limit is 130, and I usually chop around 120 .  I think my code window is about 138 characters wide, which generally avoids the need for a horizontal scrollbar.

We don't actually have a repository-wide rule, but in general it's nice to passers-by to avoid h-scroll on GitHub.